### DISCUSSION TITLE
Wake command

### DISCUSSION THREAD

**Discussion by R0GGER** [seriousm4x/UpSnap#490] [Feature requests]
Feature: Wake command. 

After waking up send a command with a delay.
The command will be executed after _n_ second(s), mintute(s), hour(s).

Delay (or ping): the command can only be executed after the server has been started.

Example:
![image](https://github.com/seriousm4x/UpSnap/assets/8298741/2f9541e0-ac12-451c-bed8-c910dd35d3df)

**Comment by seriousm4x** [comment]
I looks like you are using an outdated version of upsnap. Looks like v3.x.x but v4 is already out.

A custom wake command is already implemented but it has a different functionality as you intended. The current wake command is an alternative to the wakeonlan package being sent to wake the computer.

For any "post wake command"/"after online" stuff I'd suggest you use something like crontab on your computer which will run on startup.
